Misc/NEWS.d/next/Core and Builtins/2021-10-01-11-17-45.bpo-45329.vtqRvA.rst

```diff
@@ -0,0 +1 @@
+Fix freed memory access in :file:`Modules/pyexpat.c`
```

Comment:
It should be not in "Core and Builtins", but in "Library".

Also, `Modules/pyexpat.c` is an implementation detail and does not tell anything to common Python user. From the point of view of the Python user the bug is in the destructor of ```:class:`pyexpat.xmlparser` ```. It would be nice to add a test for this if it is possible.

Reply:
Thanks for your help! I updated the NEWS entry.